pkg/k8s/apis/cilium.io/v2/register.go

```diff
@@ -540,9 +540,37 @@ var (
 					Schema: &EndpointSelector,
 				},
 			},
+			"toGroups": {
+				Description: `ToGroups is a list of constraints that will
+				gather data from third-party providers and create a new
+				derived policy.`,
+				Properties: map[string]apiextensionsv1beta1.JSONSchemaProps{
+					"AWS": AWSGroup,
+				},
+			},
 		},
 	}
 
+	AWSGroup = apiextensionsv1beta1.JSONSchemaProps{
```

Comment:
Bump the number of the CRD validation version. (It should be on the top of this file)